src/module/module.jl

```diff
@@ -657,6 +657,21 @@ function hilbert_series(M::smodule{spoly{T}}, w::Vector{<:Integer}, shifts::Vect
   return z
 end
 
+@doc raw"""
+    dimension(M::smodule{spoly{T}}) where T <: Nemo.RingElem
+
+Given a module $M$ this function computes the Krull dimension
+of the ring $R^n/M$, where $R$ is the polynomial ring over
+which $M$ is a module and $n$ the rank. The module must be 
+given by a Groebner basis.
+"""
+function dimension(M::smodule{spoly{T}}) where T <: Nemo.RingElem
```

Comment:
Don't we us `dim` as name for this everywhere else?

Reply:
Singular.jl has already function dimension(I::sideal{spoly{T}}) where T <: Nemo.RingElem